var/ramble/repos/builtin/applications/intel-mpi-benchmarks/application.py

```diff
@@ -23,7 +23,9 @@ class IntelMpiBenchmarks(SpackApplication):
 
     name = 'IntelMpiBenchmarks'
 
-    tags = ['micro-benchmark', 'benchmark', 'mpi']
+    maintainers('rfbgo')
```

Comment:
What if I don't want to be the maintainer? ;)

Reply:
Donate it to someone else? :)